src/components/key_pad_menu/key_pad_menu_item.tsx

```diff
@@ -225,6 +231,7 @@ export const EuiKeyPadMenuItem: FunctionComponent<EuiKeyPadMenuItemProps> = ({
 
     return (
       <EuiBetaBadge
+        aria-label={betaItemLabel}
```

Comment:
This beta label can't be hard-coded because it doesn't always actually mean "Beta".

```suggestion
        aria-label={`${label} ${betaBadgeLabel}`}
```